collection-scripts/gather_crs

```diff
@@ -41,4 +45,15 @@ for resource in ${resources[@]}; do
   done
 done
 
+for ns in ${namespaces[@]}; do
+  for migplan in $(oc get migplan --namespace ${ns} --no-headers | awk '{print $1}'); do
+    mkdir -p "/must-gather/migplan-${migplan}"
+    oc get migplan ${migplan} -o yaml --namespace ${ns} &> "/must-gather/migplan-${migplan}/${migplan}.yml"
```

Comment:
We should run this oc command in async by adding `&` to the end of the line, then do a `pwait $max_parallelism` at the end of the for loop. We have had trouble with must-gather not finishing within its timeout window in the past which is why async and pwait are used.